### PR TITLE
Use date not time

### PR DESCRIPTION
**Pull request by mmazanec22**

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 # Learn more: http://github.com/javan/whenever
 
-# At minute 20 past every hour from 13 through 21 UTC.
-every '20 * * * *' do
+# At 11 UTC/7 EST every day
+every '0 23 * * *' do
```
Comment (geramirez) [inline config/schedule.rb]: I think we were going to do 7 PM, would that be 19?

File: app/parent_notifications/document_assigned_events_repository.rb [hunk 2]
```diff
@@ -14,21 +14,21 @@ class DocumentAssignedEventsRepository
...
+            WFSP.DateEntered as ExportDate,
+            -- WFSP.TimeEntered as ExportTime,
             Typ.TypeName as Source,
             Doc.IDDocuments as DocID
       from documents as Doc 
       left outer join Types as Typ on Doc.TypeId = Typ.IDTypes 
       left outer join workflowinstances as WFI on Doc.IDDocuments = WFI.docid 
       left outer join workflowsteps as WFSP on WFI.instanceID = WFSP.instancesID and WFSP.QueueEneredID in (2, 15, 17) 
-      where doc.ArchivedAt > @LastFetchedDateTime
+      where WFSP.DateEntered > @LastFetchedDate
```
Comment (geramirez) [inline app/parent_notifications/document_assigned_events_repository.rb]: so I guess this means we should set the cursor to 9/29/2019 if we want 9/30/2019 to run?

File: config/schedule.rb (hunk 1 above)
Comment (mmazanec22) [reply config/schedule.rb]: I thought this needed to be UTC?

**Comment by mmazanec22** [comment]
@geramirez re: "should we change the cursor to be a date field? I'm worried if all the timestamps are est that if a timestamp is greater or equal to 8:00pm it will set the cursor to the next day and we will miss 1 day of events. We didn't find that any documents were updated after 8 so we it might be a moot point."

Might that happen anyway if we're using dates?  Case: it's 7pm on 9/30 and we're checking against a date event cursor that says last notifications were sent 9/29.  So we're checking for any docs with DateEntered > 9/29.  We would miss anything submitted at 8 on the 29th.

If we're worried about things being added to the database later than normal working hours, we should probably instead go with your original suggestion that we check at like 11:55pm and queue notifications to be sent the next morning.

**Pull request merged**